### ISSUE TITLE
Add flag to prepend comment to file

### ISSUE THREAD

**Issue by metalmatze** [ChimeraCoder/gojson#47]
I'd like to add a comment like `// Code generated by gojson. DO NOT EDIT.` to my generated file. I don't want to do this manually.
So I propose adding `-comment` flag which would just prepend that line to the file.

**Comment by ChimeraCoder** [comment]
Yeah, this is a good idea. I actually can't think of a reason not to include this in all files (many other code generation tools in Go already do), so we may not even need to add a `-comment` flag to do so.
